analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -110,11 +111,11 @@ static ClearTextProtocolsAreSensitive()
 
             var validServerPattern = domainsList.JoinStr("|");
 
-            HttpRegex = CompileRegex(@$"^http:\/\/(?!{validServerPattern}).");
-            FtpRegex = CompileRegex(@$"^ftp:\/\/.*@(?!{validServerPattern})");
-            TelnetRegex = CompileRegex(@$"^telnet:\/\/.*@(?!{validServerPattern})");
-            TelnetRegexForIdentifier = CompileRegex(@"Telnet(?![a-z])", false);
-            ValidServerRegex = CompileRegex($"^({validServerPattern})$");
+            HttpRegex = CompileRegex(@$"^http:\/\/(?!{validServerPattern}).", overrideTimeout: RegexTimeout);
+            FtpRegex = CompileRegex(@$"^ftp:\/\/.*@(?!{validServerPattern})", overrideTimeout: RegexTimeout);
+            TelnetRegex = CompileRegex(@$"^telnet:\/\/.*@(?!{validServerPattern})", overrideTimeout: RegexTimeout);
+            TelnetRegexForIdentifier = CompileRegex("Telnet(?![a-z])", false);
+            ValidServerRegex = CompileRegex($"^({validServerPattern})$", overrideTimeout: RegexTimeout);
```

Comment:
This is very painful change that doens't make much sense. Just use the higher timeout everywhere instead.